pages/documentation/index.js

```diff
@@ -0,0 +1,26 @@
+import Header from "../../components/header/header";
+import Footer from "../../components/footer/footer";
+import styles from "./documentation.module.css";
+
+// /* eslint-disable max-len */z
+
+export default function Documentation() {
```

Comment:
From `About Us`

```Our open source explorer analyzes your projects `package.json` file, and returns a list of projects that are being used as a first or second-degree dependency that may need some attention from fellow engineers (you!). In other words, projects that are in active development yet fall below the `developer fame` threshold.```

Reply:
I don't think we need extra page for documentation.
The point is we don't have a good explanation for our product.

I would work on current About Us page content to explain the product.
And maybe it needs to be renamed to `About` (And have a section both about US and Product)